AirCasting/Settings/SettingsView.swift

```diff
@@ -12,7 +12,9 @@ struct SettingsView: View {
     let urlProvider: BaseURLProvider
     let logoutController: LogoutController
     @State private var showModal = false
+    @State private var isShowingLoadingScreen = false
```

Comment:
same - can't we use `userState` directly instead?

Reply:
Yes we can!